esileapclient/tests/unit/osc/v1/test_offer.py

```diff
@@ -114,25 +114,21 @@ def test_offer_list(self):
 
         collist = [
             "UUID",
-            "Name",
-            "Lessee ID",
+            "Resource",
```

Comment:
Resource just replaces Name right?

Reply:
Not quite - "Name" is the offer name. I'm removing it from the default list view because it's optional and I'm not sure people will find it as useful as I thought.